de-DE/docs/script.nhlp

```diff
@@ -24,10 +24,14 @@
 		<keyword>inclu</keyword>
 		<keyword>script</keyword>
 		<keyword>start</keyword>
+		<keyword>from line</keyword>

```

Comment:
Whitespaces do not work here. Use the new option to enable searching for the entry